libbeat/tests/system/requirements.txt

```diff
@@ -2,3 +2,5 @@ nose
 jinja2
 PyYAML
 nose-timer
+pip
```

Comment:
Yes , il will update the outdated version of pip which is shipped with virtualenv